src/main/kotlin/org/jitsi/rtp/rtcp/RtcpSdesPacket.kt

```diff
@@ -182,9 +185,11 @@ abstract class SdesItem(
 
 class UnknownSdesItem(
     private val sdesTypeValue: Int,
-    private val dataField: ByteArray
+    buf: ByteArray,
+    offset: Int,
+    length: Int
 ) : SdesItem(SdesItemType.UNKNOWN) {
-
+    private val dataField = copyData(buf, offset, length)
```

Comment:
Why do we have to copy here if we didn't before?

Reply:
We did have a copy before, but it was created outside this class. Sortbugs warned against it because this class' data was exposed.

Reply:
I think we've established that packets 'own' the buffer they are created with (at least starting at `offset` for `length`), so this seems to go against that.

Reply:
True this isn't a `Packet`, but I think this is generally the m.o. in `jitsi-rtp` (but maybe I'm misremembering).  I think any packet would then have this same complaint from spotbugs, right? They all take a buffer and expose it via getBuffer?

Reply:
The way the test is written only the base class (the one that exposes a reference to an array) triggers a warning.